webapp/src/index.tsx

```diff
@@ -361,6 +373,15 @@ export default class Plugin {
             );
         };
 
+        const followThread = async (channelID: string, teamID: string) => {
+            const threadID = voiceChannelRootPost(store.getState(), channelID);
+            if (threadID) {
+                setThreadFollow(getCurrentUserId(store.getState()), teamID, threadID, true)(store.dispatch, store.getState);
```

Comment:
Nit: we could probably do this, right?
```suggestion
                store.dispatch(setThreadFollow(getCurrentUserId(store.getState()), teamID, threadID, true));
```